Contributions/Using_Linux_in_a_Safe_System.md

```diff
@@ -0,0 +1,486 @@
+**Premise:** some of the following considerations assume a fairly high level
+of freedom in designing the overall system.
+While one must acknowledge that in practice such high level of freedom will 
+not always be available, due to design constraints, it is equally true that
+attempting to cope with constraints can lead to an even higher total cost of
+ownership. 
+
+
+* **Clearly define requirements toward system integrity and availability, upfront**
+
+  Clarifying the purpose and the scope of the design goes a long way in defining criteria for
+  identifying acceptable solutions.
+
+  Strict requirements on availability can drastically alter the system design.
+
+  For example, having a requirement to simply report failures within a maximum
+  allotted time, is usually significantly simpler than having a stricter requirement
+  to guarantee that a certain functionality will not be interrupted.
+
+  Therefore, the intended use-cases play a major role in curbing potential
+  system designs.
+
+* **Linux is not intrinsically safe**
+
+  While Linux has many remarkable properties, safety is not one of them, which means
+  that it won't be possible to just allocate safety requirements to Linux, and expect
+  them to be automatically satisfied. Quite the opposite, actually.
+
+  Having to cope with safety requirements assigned to Linux will involve a price to
+  be paid, in one form of another.
+
+  The problem with said cost is that it might *also* affect negatively the goals
+  that led to the selection of Linux, in the first place. For example, if Linux was
+  selected because of its a rapid turnaround in fixing security bugs, selecting
+  an approach that would introduce an lengthy qualification process would undermine
+  the advantage of the rapid turnaround.
+
+* **Aggressively placing safety requirements on anything but (components of) the Linux kernel**
+
+  The natural consequence of the previous point is that - given a specific set of
+  safety goals for the system as a whole - one should explore if components other
+  than Linux could be the recipient of the ensuing safety requirements
+  (more on this later).
+
+  There are several additional reasons for favouring an external component, besides
+  what has been mentioned already:
+  *  much greater probability for the component to be simpler (and thus easier to qualify)
+  *  opportunity to have a better control over the component (e.g. use a software
+     architecture that is better suited for safety, formalised processes, etc.)
+  *  decouple release cycles of the Linux kernel from the safety component
+  *  other non-technical reasons (e.g. different licensing)
+
+  A specific application or use case, will have a broader set of requirements,
+  like Total Cost of Ownership, Mean Time Before Failure, weight, repairability,
+  just to mention a few, and this can and will affect the decision of what's the
+  optimal compromise between allocating specific safety requirements on either
```

Comment:
```suggestion
  just to mention a few, and this can and will affect the decision of what constitutes
  an optimal compromise between allocating specific safety requirements on either
```